charts/datadog/templates/_containers-init-linux.yaml

```
@@ -1,5 +1,6 @@
 {{- define "containers-init-linux" -}}
 - name: init-volume
+{{ include "generate-security-context" (dict "securityContext" .Values.agents.containers.initContainers.securityContext "targetSystem" .Values.targetSystem "seccomp" "" "kubeversion" .Capabilities.KubeVersion.Version) | indent 2 }}
```

Adding a custom `securityContext` does not seem to be supported in GKE Autopilot. Can you please add a condition here to check if `.Values.providers.gke.autopilot` is not enabled?

```suggestion
{{- if not .Values.providers.gke.autopilot }}
{{ include "generate-security-context" (dict "securityContext" .Values.agents.containers.initContainers.securityContext "targetSystem" .Values.targetSystem "seccomp" "" "kubeversion" .Capabilities.KubeVersion.Version) | indent 2 }}
{{- end }}
```